wpiutil/src/main/java/edu/wpi/first/util/MsvcRuntimeException.java

```diff
@@ -21,7 +21,7 @@ private static String generateMessage(
                 expectedMajor, expectedMinor, foundMajor, foundMinor))
         .append(String.format("JVM Location: %s\n", jvmLocation))
         .append(String.format("Runtime DLL Location: %s\n", runtimePath))
-        .append("See https://wpilib.org/jvmruntime for more information\n");
+        .append("See https://docs.wpilib.org/jvmruntime for more information\n");
```

Comment:
This link doesn't work for me either. We usually add redirects for links like wpilib.org/stacktrace.

Reply:
The article doesn't exist yet. And I thought redirects were only a thing for docs.wpilib.org. But if the redirect can be added to wpilib.org, then this PR is unnecessary.